NPM package not usable

I tried to install this very helpful instanced-mesh package via npm, but got an error when attempting to use it.

Version used (from `package.json`):

```
   "aframe-instanced-mesh": "^0.7.3",
```

Import clause:

```
import "aframe-instanced-mesh";
```

Error message:

```
X [ERROR] Could not resolve "./instanced-mesh.js"

    node_modules/aframe-instanced-mesh/index.js:1:24:
      1 │ export { default } from './instanced-mesh.js';
        ╵                         ~~~~~~~~~~~~~~~~~~~~~
```

Most probably, the problem is in the location of `index.js` and `instanced-mesh.js`, which are not in the same directory.

--- a/index.js
+++ b/index.js
@@ -1,1 +1,0 @@
-export { default } from './instanced-mesh.js';

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "aframe-instanced-mesh",
   "version": "0.7.3",
   "description": "Use three.js InstancedMesh object within A-Frame",
-  "main": "index.js",
+  "main": "src/instanced-mesh.js",
   "directories": {
     "test": "tests"
   },

